index.bs

```diff
@@ -10317,16 +10317,13 @@ of {{AudioParamDescriptor}}s.
 		every <a>render quantum</a>, if {{AudioWorkletNode}} is
 		[=actively processing=].
 
-		The return value of this method controls the lifetime of the
-		{{AudioWorkletProcessor}}'s associated
-		{{AudioWorkletNode}}. At the conclusion of each call to the
-		{{process()}} method, <a href="https://tc39.github.io/ecma262/#sec-toboolean"><code>ToBoolean</code></a> (described in [[!ECMASCRIPT]])
-		is applied to the return value and the result is assigned to the associated AudioWorkletProcessor's <a>active source</a> flag. This in turn affects whether subsequent
-		invocations of {{process()}} occur, and has an impact on the lifetime of the node.
-
 		<div class="note">
-			This lifetime policy can support a variety of approaches found in
-			built-in nodes, including the following:
+			The return value of this method controls the lifetime
+			of the {{AudioWorkletProcessor}}'s associated
+			{{AudioWorkletNode}}.
+
+			This lifetime policy can support a variety of approaches
+			found in built-in nodes, including the following:
```

Comment:
If you do this, then where is the normative description of what the return value means?

Reply:
Then perhaps we want to put l.10321-10323 back to where it was?

Reply:
Oops. I missed the other part.  lgtm.